src/router.sh

```diff
@@ -19,15 +19,16 @@ echo "${stage} stage selected"
 if [ "${platform}" = "go" ]; then
     executable_script="${SCRIPT_DIR}go-chaincode/${stage}.sh"
     echo "Go selected"
-    echo ${executable_script}
+    echo "$executable_script"
 elif [ "${platform}" = "composer" ]; then
     executable_script="${SCRIPT_DIR}composer/${stage}.sh"
     echo "Composer selected"
-    echo ${executable_script}
+    echo "$executable_script"
 else 
     echo "Invalid platform: ${platform} selected"
     exit 1
 fi
 
 # ok now let's call the command we are routing to
-source ${executable_script}
+# shellcheck disable=1090
```

Comment:
Makes sense not to worry about this file, although [the shellcheck doc](https://github.com/koalaman/shellcheck/wiki/SC1090) seems to suggest pointing to `/dev/null` rather than skipping the check...

> If you don't care that ShellCheck is unable to account for the file, specify `# shellcheck source=/dev/null`.

Either way, it would be good to include a comment about why this check can safely be ignored, i.e. the script is intended to select the correct file to source, which is tested elsewhere.

Reply:
Hmm, yes redirecting to null seems to be the SC1090 specific way to fix. I was using more generic.